v8/config/krb5conf.go

```diff
@@ -726,3 +731,100 @@ func (c *Config) JSON() (string, error) {
 	}
 	return string(b), nil
 }
+
+var expandMap map[string]string
```

Comment:
I have been trying to avoid global variables. Could this be a local variable to the LibDefaults.parseLines() method function?